### PR TITLE
Docs: Build on Summit (OLCF) with CMake

### PR DESCRIPTION
**Pull request by LDAmorim**
Hi all,

This tiny PR changes the `Docs` instructions for building *WarpX* on SUMMIT@OLCF to the `cmake` commands.

As suggested by @ax3l in issue #1453 .

The file changed is:
* Docs/source/building/summit.rst - to link to the `cmake` `Docs` page and to change the compiling commands.

ps: I am still installing sphinx on my "new" Windows 10 + ubunto bash shell machine.
So I will update you with the confirmation that there are no visualization/linking errors in a following message.

Cheers,
Diana

### CODE REVIEW

**Review by mrowan137 — APPROVED** [review]
Thanks for this!  LGTM.

File: Docs/source/building/summit.rst [hunk 1]
```diff
@@ -130,11 +130,14 @@ Optionally, download and install :ref:`libEnsemble <libensemble>` for dynamic en
    python3 -m pip install --upgrade mpi4py --no-binary mpi4py
    python3 -m pip install --upgrade -r $HOME/src/warpx/Tools/LibEnsemble/requirements.txt
 
-Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following commands to compile:
+Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following commands to compile with :ref:`_building-cmake`:
```
Comment (ax3l) [inline Docs/source/building/summit.rst]: Sorry for this, I meant the link below the command :)
```suggestion
Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following commands to compile:
```

File: Docs/source/building/summit.rst [hunk 2]
```diff
@@ -130,11 +130,14 @@ Optionally, download and install :ref:`libEnsemble <libensemble>` for dynamic en
...
 
 .. code-block:: bash
 
-   make -j 16 COMP=gcc USE_GPU=TRUE USE_OPENPMD=TRUE
+   mkdir -p build
+   cd build
+   cmake .. -DWarpX_OPENPMD=ON -DWarpX_DIMS=3 -DWarpX_COMPUTE=CUDA -DCMAKE_CUDA_ARCHITECTURES=70 -DCUDA_ARCH=7.0
+   make -j 10
 
 The other :ref:`general compile-time options <building-source>` apply as usual.
```
Comment (ax3l) [inline Docs/source/building/summit.rst]:
```suggestion
The general :ref:`cmake compile-time options <building-cmake>` apply as usual.
```

File: Docs/source/building/summit.rst (hunk 1 above)
Comment (LDAmorim) [reply Docs/source/building/summit.rst]: Thanks @ax3l
Oh no need to be sorry! It makes more sense that way.

File: Docs/source/building/summit.rst [hunk 3]
```diff
@@ -134,9 +134,12 @@ Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following comman
 
 .. code-block:: bash
 
-   make -j 16 COMP=gcc USE_GPU=TRUE USE_OPENPMD=TRUE
+   mkdir -p build
+   cd build
+   cmake .. -DWarpX_OPENPMD=ON -DWarpX_DIMS=3 -DWarpX_COMPUTE=CUDA -DCMAKE_CUDA_ARCHITECTURES=70 -DCUDA_ARCH=7.0
```
Comment (ax3l) [inline Docs/source/building/summit.rst:139]: Thank you!

With the (not yet merged) PR #1459 this will become shorter again. Will update once this landed.

**Review by ax3l — APPROVED** [review]

**Pull request merged**